src/Controller/PeopleController.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace App\Controller;
+
+use Exception;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class PeopleController extends AbstractController
+{
+    /**
+     * @Route("/people", name="people")
+     */
+    public function index()
+    {
+        return $this->render('people/index.html.twig', [
+            'controller_name' => 'PeopleController',
+        ]);
+    }
+
+    /**
+     * @Route(
+     *     "/validate/{element}",
+     *     name="validatePerson",
+     *     requirements={"POST"}
+     * )
+     */
+
+    public function validate(Request $request, string $element)
+    {
+        try {
+            $input = json_decode($request->getContent(), true)['input'];
+        } catch (Exception $e) {
+            return new JsonResponse(['error' => 'Invalid method'], Response::HTTP_BAD_REQUEST);
+        }
+        $students = $this->getStudents();
+        $projects = $this->getProjects();
+
+        switch ($element) {
+            case 'name':
+                return new JsonResponse(['valid' => in_array(strtolower($input), $students)]);
+            case 'project':
+                return new JsonResponse(['valid' => in_array(strtolower($input), $projects)]);
+        }
+        return new JsonResponse(['error' => 'Invalid method'], Response::HTTP_BAD_REQUEST);
+    }
+
+    private function getStorage()
+    {
+        $studentsJson = file_get_contents('students.json');
+        return $studentsJson;
+
+    }
+    private function getStudents(): array
+    {
+        $students = [];
+        $storage = json_decode($this->getStorage(), true);
+        foreach ($storage as $teamData) {
+            foreach ($teamData['students'] as $student) {
+                $students[] = strtolower($student);
+            }
+        }
+        return $students;
+    }
+
+    private function getProjects(): array
+    {
+        $projects = [];
+        $storage = json_decode($this->getStorage(), true);
+
+        foreach ($storage as $key => $value) {
```

Comment:
Dabar veikia, bet gal praverst požiūrio praplėtimui.
```suggestion
        foreach (array_keys($storage) as $key) {
```
Su `array_keys` funkcija iš karto gaunami visos `keys` reikšmės, tada nebereikia naudoti `$value` kintamojo, kuris jokios naudos neduoda.

Plačiau:
 * https://www.php.net/manual/en/function.array-keys.php